scripts/constants.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+PACKAGES = [
```

Comment:
Add a comment about how these packages are selected (what the category for them being aggregated)?

Reply:
Perhaps using `pathlib` to construct correct paths based on this file's location instead of string literals.

Reply:
I plan to change them via checking if there is a `Cargo.toml` in the dir. But I want to refactor this part in future PRs. This PR is large enough.